### ISSUE TITLE
STL with holes in it

### ISSUE THREAD

**Issue by gitKlaasTijsen** [asmorgan24/pngSlicer#6]
hello,

I stumbled across this project, and not sure if there is development going on with it but was testing with a range of stl files and just found something I'm not sure if this isn't implemented yet or something that's not possible.

but basically I have a tube-like stl shape, yet the created png files don't have a hole in the middle. and I guess this is super important as many stl have holes. and im not sure if that's something that should be already working or not.

the stl I tried -> [https://www.thingiverse.com/thing:492067/files](url)

other png slicer program img [https://ibb.co/3NVkjSs](url)

thanks :)

**Comment by asmorgan24** [comment]
Yeah, this is not supported. This project was just more of a fun project
that I never really went anywhere with. Feel free to contribute if you
would like. :)

-Andy

**Comment by gitKlaasTijsen** [comment]
ok, thanks I will attempt haha. but I thought I would still ask if you had any idea how you would actually allow for holes within STL,

I never really dealt with python imagine manipulation so not super in-depth with this

- I attempted to set change draw.Polygon to draw.point and it does seem it can see the different circles [https://ibb.co/tq7YGYm](url)  (kinda hard to see but they are there) but other than that I'm not too sure haha

thanks :)
